web_poet/page_inputs/browser.py

```diff
@@ -0,0 +1,10 @@
+from web_poet.mixins import SelectableMixin
+
+
+class BrowserHtml(str, SelectableMixin):
```

Comment:
:lipstick: It should not be a problem given the current interface, but I would expect:

```suggestion
class BrowserHtml(SelectableMixin, str):
```

Reply:
nice catch!